keyboards/pizzakeyboards/slice65/readme.md

```diff
@@ -0,0 +1,27 @@
+# Pizza Keyboards (V2) Slice65
+
+![Pizza V2](https://i.imgur.com/RfqnpVl.jpeg)
```

Comment:
```suggestion
![Pizza V2](https://i.imgur.com/RfqnpVlh.jpeg)
```

Reply:
I prefer the 1200px wide version of this image.  It's only 45k more and adds quite a bit of detail.  Do I have to make this change in order to get this keyboard added?

Reply:
Check PR guidelines regarding images in `readme.md` files 😉